cmd/releasego/testdata/publish-announcement/three-branches.golden.md

```diff
@@ -1,7 +1,7 @@
 ---
-post_title: Go 1.23.1-1, 1.22.8-1, and 1.21.11-16 Microsoft builds now available
+post_title: Go 1.22.8-1, 1.23.1-1, and 1.21.11-16 Microsoft builds now available
```

Comment:
It looks like the logic in `generateBlogPostTitle` actually mangles the sort a little.